app/api/currentuser/current_user.go

```diff
@@ -33,6 +33,7 @@ func (srv *Service) SetRoutes(router chi.Router) {
 	router.Post("/current-user/group-requests/{group_id}", service.AppHandler(srv.sendGroupRequest).ServeHTTP)
 
 	router.Get("/current-user/group-memberships", service.AppHandler(srv.getGroupMemberships).ServeHTTP)
+	router.Post("/current-user/group-memberships", service.AppHandler(srv.joinGroupByPassword).ServeHTTP)
```

Comment:
As discussed in the doc, probably it should be `POST /current-user/group-memberships/by-password`

Reply:
fixed